client/signup/steps/domains/index.jsx

```diff
@@ -319,7 +341,7 @@ class DomainsStep extends React.Component {
 					onRegisterDomain={ this.handleAddDomain }
 					onMapDomain={ this.handleAddMapping.bind( this, 'mappingForm' ) }
 					onSave={ this.handleSave.bind( this, 'mappingForm' ) }
-					products={ productsList.get() }
+					products={ this.props.productList }
```

Comment:
Not sure why, but I got a failure, but seems the product list is missing:
```
Warning: Failed prop type: The prop `products` is marked as required in `MapDomainStep`, but its value is `undefined`.
```

Which led to a total failure of signup:
```
index.jsx:199 Uncaught TypeError: Cannot read property 'domain_map' of undefined
```

I've tried fixing that in https://github.com/Automattic/wp-calypso/pull/26942/commits/eaa6c9dfff0fee28be827428edab7ef3e3a2c4cd, but now the price of the domain mapping on the MapDomainStep shows up as `Free` always 🤦‍♂️

Reply:
2 suggestions.

One is to see what `DomainPriceRule` does, and skio price if nothing passed. 

Otherwise, add placeholders for the component while waiting for the price list.

Reply:
That's the thing - it never changes from `Free` to the correct price. Not sure why's that 🤔

Reply:
Ow, ow. Maybe the format of the store and the one from the query component aren't the same and it needs adjusting?

Reply:
Nope. It's interchangeable. https://github.com/Automattic/wp-calypso/pull/23327

Well, there might be a bug then. Not sure why it happens unless we're passing the productList wrongly.

Reply:
Easy fix... I used productList insteas of productsList when assigning the props var. Eh. Mobile debugging ftw. :laughing: